### ISSUE TITLE
VCS 4.0.34: Unhandled error when exporting missing linked table

### ISSUE THREAD

**Issue by HughWarrington** [joyfullservice/msaccess-vcs-addin#500]
```
-------------------------------------
my_database.mdb
VCS Version 4.0.34
Performing Full Export
22/03/2024 13:30:09
-------------------------------------
```

Running with debug enabled.

![image](https://github.com/joyfullservice/msaccess-vcs-addin/assets/5960770/f7568466-5970-4690-8749-cc131c0c2a2e)

![image](https://github.com/joyfullservice/msaccess-vcs-addin/assets/5960770/4b2bb74c-f965-4c42-9390-b3646c377dd5)

I think the screenshots help explain what's going on.

It's trying to export data from a table. `clsDbTableData.GetTableExportSql` has generated a select query that is missing the `ORDER BY` columns at the end. I'm guessing this is because the table in question is a linked table that cannot be found on disk.

I'm not sure what the conventions are in this project (or in Access/VB in general) but perhaps we should instead log a meaningful warning saying 'missing linked table' or similar.

**Comment by joyfullservice** [comment]
Yes, it looks like we should test to ensure the table exists and is available before attempting an export of the data. Good idea!

**Comment by joyfullservice** [comment]
Added some code to ensure the availability of the target for linked tables, and logs the error more clearly for the user.

![image](https://github.com/joyfullservice/msaccess-vcs-addin/assets/7254992/cbb4f70a-e1a7-4fc1-a298-ab274f21210b)

**Comment by HughWarrington** [comment]
Brilliant! Thank you.

**Comment by hecon5** [comment]
If this is working @HughWarrington, can you close it? Thank you!
